ts/screens/wallet/AddPaymentMethodScreen.tsx

```diff
@@ -58,7 +60,8 @@ const contextualHelpMarkdown: ContextualHelpPropsMarkdown = {
 
 const getpaymentMethods = (
   props: Props,
-  onlyPaymentMethodCanPay: boolean
+  onlyPaymentMethodCanPay: boolean,
+  isPaymentOnGoing: boolean = false
```

Comment:
Having more than one boolean parameter makes functions less readable in my experience.
I suggest you go for an object like: 

```
const getpaymentMethods = (
  props: Props,
  options: {
      onlyPaymentMethodCanPay: boolean,
      isPaymentOngoing: boolean,
  }
) => {
```

Reply:
Yes, I got the same feeling!

I accepted the suggestion
bc9f2c6ff4efd2c8cfd71890848c7bf7ba2c7a90